docs/ref/models/fields.txt

```diff
@@ -1550,6 +1550,31 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
     integrity, this will cause an :exc:`~django.db.IntegrityError` unless
     you manually add an SQL ``ON DELETE`` constraint to the database field.
 
+* .. attribute:: DB_CASCADE
+
+    .. versionadded:: 3.1
+
+    Django takes no direct action, behaves exactly like DO_NOTHING,
+    but also sets ON DELETE CASCADE as a SQL constraint. The cascade
```

Comment:
Please wrap the SQL `ON DELETE *` in double backticks in each of these new sections.